app/assets/javascripts/backbone/views/breakeven_view.coffee

```diff
@@ -0,0 +1,39 @@
+class MS.Views.BreakevenView extends Backbone.View
+
+  # The view is called from the loan questionnare view.
+
+  events:
+    'change .editable-table[data-table="fixed_costs"] input.amount': 'totalFixedCostsChanged'
+
+  initialize: (e) ->
+    @$(".editable-breakeven-table").each (index, table) =>
+      new MS.Views.EditableBreakevenTableView(el: table)
+
+    @products = @$("tr[data-group='product']:not('.hidden')").map (index, productRow) =>
+      new MS.Views.BreakevenProductView(el: productRow)
+
+    totals = @$("tr[data-group='product-total']").map (index, totalRow) =>
+      new MS.Views.BreakevenProductTotalView(el: totalRow, products: @products)
+
+    # Should be 1 and only 1 total row
+    @total = totals[0]
```

Comment:
This seems an odd way to do things if there is only 1. Why not use jQuery's `.first()` above? `@total = new MS.Views.BreakevenProductTotalView(el: @$("tr[data-group='product-total']").first(), products: @products)`